File: dataset/codenet/Python/Code/s246280701.py
n = int(input())
x = list(map(int, input().split()))
ans = 10**10
for i in range(101):
    tmp = [abs(i - x[j])**2 for j in range(n)]
    if sum(tmp) < ans:
        ans = sum(tmp)

print(ans)

File: output/gpt_4o/codenet/direct_translation/python/java/s246280701.java

import java.util.Scanner;

public class s246280701 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int[] x = new int[n];
        for (int j = 0; j < n; j++) {
            x[j] = scanner.nextInt();
        }
        long ans = (long)Math.pow(10, 10);
        for (int i = 0; i <= 100; i++) {
            long sum = 0;
            for (int j = 0; j < n; j++) {
                sum += Math.pow(Math.abs(i - x[j]), 2);
            }
            if (sum < ans) {
                ans = sum;
            }
        }
        System.out.println(ans);
    }
}

